clamav.py

```diff
@@ -195,21 +197,48 @@ def scan_file(path):
         stdout=subprocess.PIPE,
         env=av_env,
     )
-    output = av_proc.communicate()[0].decode()
-    print("clamdscan output:\n%s" % output)
 
-    # Turn the output into a data source we can read
-    summary = scan_output_to_json(output)
+    try:
+        output, errors = av_proc.communicate(timeout=CLAMDSCAN_TIMEOUT)
+    except subprocess.TimeoutExpired:
+        av_proc.kill()
+        output, errors = av_proc.communicate()
+
+    decoded_output = output.decode()
```

Comment:
The previous code took the first element from `communicate()` and decoded it. 
Is it intentional that we're decoding the entire output response here?